extensions/aria_extension_tosca/simple_v1_0/modeling/capabilities.py

```diff
@@ -139,9 +139,18 @@ def get_template_capabilities(context, presentation):
                 if values:
                     capability_assignment._raw['properties'] = values
                     capability_assignment._reset_method_cache()
+
```

Comment:
package requires explaining

Reply:
It's explained in the `__init__.py` of the package:

```
"""
Creates ARIA service template models based on the TOSCA presentation.

Relies on many helper methods in the presentation classes.
"""
```